raiden/tests/integration/transfer/test_mediatedtransfer.py

```diff
@@ -101,3 +102,42 @@ def test_mediated_transfer_with_entire_deposit(
         func=assert_synched_channel_state,
         timeout=network_wait,
     )
+
+
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('number_of_nodes', [3])
+def test_mediated_transfer_too_many(raiden_network, token_addresses):
```

Comment:
do we really need an integration test for this? can't it be just an unit test?

Reply:
Yes, I replaced it with a unit test.